installer/dbscript/dbscript-2.sql

```diff
@@ -4343,7 +4351,13 @@ VALUES (1, 0, 1, 0, 0, @apiv2_time_time_sheet_config_data_group_id, @admin_role_
        (1, 0, 0, 0, 0, @apiv2_time_employee_timesheet_list_data_group_id, @admin_role_id),
        (1, 0, 0, 0, 0, @apiv2_time_employee_timesheet_list_data_group_id, @supervisor_role_id),
        (1, 0, 0, 0, 0, @apiv2_time_default_timesheet_data_group_id, @admin_role_id),
-       (1, 0, 0, 0, 0, @apiv2_time_default_timesheet_data_group_id, @ess_role_id);
+       (1, 0, 0, 0, 0, @apiv2_time_default_timesheet_data_group_id, @ess_role_id),
+       (1, 0, 0, 0, 0, @apiv2_time_reports_data_group_id, @admin_role_id),
+       (1, 0, 0, 0, 0, @apiv2_time_reports_data_group_id, @project_admin_role_id),
+       (1, 0, 0, 0, 0, @apiv2_time_reports_data_data_group_id, @admin_role_id),
+       (1, 0, 0, 0, 0, @apiv2_time_reports_data_data_group_id, @project_admin_role_id),
+       (1, 0, 0, 0, 0, @time_report_project_report_data_group_id, @admin_role_id),
+       (1, 0, 0, 0, 0, @time_report_project_report_data_group_id, @project_admin_role_id);
```

Comment:
will remove this 2 lines, since already defined under `time_project_reports`

Reply:
done